guides/coding_style/c++/README.md

```diff
@@ -155,15 +156,15 @@ void foo()
     else
         res &= false;
 
-    if ((1 != 0)
-        && (1 == true || 0 == false))
+    if ((1 != 0) &&
+        (1 == true || 0 == false))
         res &= true;
     else
         res &= false;
 
-    if (res != true
-        || (!(res == true)
-            || (res == false)))
+    if (res != true ||
+         (!(res == true) ||
+            (res == false)))
```

Comment:
In my opinion, the conditions must be aligned to previous line.
i.e., in this situation, the code will be:
```
    if (res != true ||
        (!(res == true) ||
         (res == false)))
```

Reply:
If there used parentheses then align with the char after open parenthesis.